clap_markdown/src/lib.rs

```diff
@@ -0,0 +1,204 @@
+            let _ = writeln!(markdown, "{description}");
+        }
+
+        // Add usage example.
+        let usage = cmd.render_usage().to_string();
+        let _ = writeln!(markdown, "\n```sh\n{usage}\n```");
+
+        // Add positional arguments.
+        let mut positionals = cmd.get_positionals().flat_map(generate_argument).peekable();
+        if positionals.peek().is_some() {
+            let _ = writeln!(markdown, "\n### Arguments");
+        }
+        for positional in positionals {
+            let _ = writeln!(markdown, "\n{positional}");
+        }
+
+        // Add options.
+        let mut options = cmd
+            .get_arguments()
+            .filter(|arg| !arg.is_positional())
+            .flat_map(generate_argument)
+            .peekable();
+        if options.peek().is_some() {
+            let _ = writeln!(markdown, "\n### Options");
+        }
+        for option in options {
+            let _ = writeln!(markdown, "\n{option}");
+        }
+
+        // Add subcommands.
+        let mut subcommands = cmd
+            .get_subcommands()
+            .filter(|cmd| !cmd.is_hide_set() && cmd.get_name() != "help")
+            .peekable();
+        if subcommands.peek().is_some() {
+            let _ = writeln!(markdown, "\n### Commands\n");
+        }
+        for cmd in subcommands {
+            let name = cmd.get_name();
+            let human_path = format!("{} {name}", parents.join(" "));
+            let link_path = format!("{}_{name}", parents.join("_"));
+            let _ = writeln!(markdown, "* [{}](./{})", human_path, link_path);
+        }
+
+        Self { command: parents, content: markdown }
+    }
+}
+
+/// Convert argument to markdown.
+fn generate_argument(arg: &Arg) -> Option<String> {
+    // Don't show hidden arguments.
+    if arg.is_hide_set() {
+        return None;
+    }
+
+    let mut markdown = String::new();
+
+    // Add short option.
+    if let Some(short) = arg.get_short() {
+        let _ = write!(markdown, "-{short}");
+    }
+
+    // Add long option.
+    if let Some(long) = arg.get_long() {
+        if !markdown.is_empty() {
+            markdown += ", ";
+        }
+        let _ = write!(markdown, "--{long}");
+    }
+
+    // Add arguments.
+    let min_required = arg.get_num_args().map_or(0, |num| num.min_values());
+    let all_optional = arg.is_positional() && !arg.is_required_set();
+    if let Some(value_names) = arg.get_value_names() {
+        if !markdown.is_empty() {
+            markdown += " ";
+        }
+
+        let delimiter = arg.get_value_delimiter().unwrap_or(' ');
+
+        for (i, value_name) in value_names.iter().enumerate() {
+            // Add separator between parameters.
+            if i != 0 {
+                markdown.push(delimiter);
+            }
+
+            if i >= min_required || all_optional {
+                let _ = write!(markdown, "[{value_name}]");
+            } else {
+                let _ = write!(markdown, "<{value_name}>");
```

Comment:
```suggestion
                let _ = write!(markdown, "`[{value_name}]`");
            } else {
                let _ = write!(markdown, "`<{value_name}>`");
```
To make the positional arguments stand out in the docs.